cmd/keymasterd/config.go

```diff
@@ -204,6 +227,39 @@ func loadVerifyConfigFile(configFilename string) (RuntimeState, error) {
 		logger.Debugf(3, "client ca file loaded")
 
 	}
+	if len(runtimeState.Config.Base.KeymasterPublicKeysFilename) > 0 {
+		filename := runtimeState.Config.Base.KeymasterPublicKeysFilename
+		if _, err := os.Stat(filename); os.IsNotExist(err) {
+			logger.Printf("keymaster_public_keys_filename defined but file does not exist")
+			return runtimeState, err
+		}
```

Comment:
I think it's cleaner to move the `IsNotExist` test into the error clause below when you open the file. In other words, instead of the "check and then open", just do "open and check error type". For a later PR.